packages/distros/rke2/zarf.yaml

```diff
@@ -0,0 +1,49 @@
+kind: ZarfInitConfig
```

Comment:
Since this is an `init` config, the internal logic is looking for an injector at a hardcoded path of `src/injector/zarf-registry` which only works if the zarf definition is at the root level of this project. 

That said, that' a bug on the CLI side and not a bug in this PR. @jeff-mccoy

Reply:
Do we want this to be an init config? I'm thinking no, that it should be a package config

Reply:
Yeah we do have an example of this now using `ZarfPackageConfig`: https://github.com/defenseunicorns/zarf/blob/master/packages/distros/eks/zarf.yaml.